pelican/plugins/test/test_show_source.py

```diff
@@ -0,0 +1,30 @@
+import pathlib
+
+import pytest
+
+import pelican
+
+
+@pytest.fixture(scope="module", autouse=True)
+def site(tmp_path_factory):
+    output = tmp_path_factory.getbasetemp()
+    datadir = pathlib.Path(__file__).parent / "data"
+    args = [
+        datadir / "content",
+        "-o",
+        output,
+        "-s",
+        datadir / "pelicanconf.py",
+        "--relative-urls",
+    ]
+    pelican.main([str(a) for a in args])
+    yield output
+
+
+def test_show_source(site):
+    tag = "esse-quam-laboriosam-at-accusantium"
+
+    assert (site / (tag + ".html")).exists()
```

Comment:
Another `exists` check for the source (`.txt`) here?

Reply:
one to test the generated file location and another for the content. Where would be the other exists check?

Reply:
You check that the content has a link to it (below) but you don't check if the source itself is copied over:

```python
assert (site / (tag + ".txt")).exists()
```

Reply:
thanks, I've update the test to include the check for the source file (and put some comment in the code)

Reply:
thanks